tools/text_processing_deployment/alignment.py

```diff
@@ -0,0 +1,165 @@
+import os
+import string
+from argparse import ArgumentParser
+from typing import List
+
+import pynini
+from pynini import Far
+
+from nemo.utils import logging
+
+
+"""
+This files takes 1. FST path 2. entire string 3. start position of substring 4. end (exclusive) position of substring
```

Comment:
could you please specify what fst graph is expected, i.e. en_tn_True_deterministic_cased__tokenize.far